### ISSUE TITLE
The exact Name `e_ac8q' is not in scope

### ISSUE THREAD

**Issue by pcarbonn** [haskus/haskus-utils#5]
When I create the following node definition:

    data AddF e = AddF e (e, e) deriving (Functor)
    eadtPattern 'AddF "Add"

I get the following error message:

```
    The exact Name `e_ac8q' is not in scope
      Probable cause: you used a unique Template Haskell name (NameU),
      perhaps via newName, but did not bind it
      If that's it, then -ddump-splices might be useful
   |
14 |   eadtPattern 'AddF "Add"
   |   ^^^^^^^^^^^^^^^^^^^^^^^
```

I suspect it is because of the tuple (e,e).

**Comment by hsyl20** [comment]
Aha! I'm writing an announce for the new release and I just stumbled on this bug too while writing an example!

It should be fixed by: 48b1c18006207c1c8a6168afb059d65b8e07ff05 I will release a 2.1...

**Comment by hsyl20** [comment]
I've just released 2.0.1 with the fix. Reopen if it still fails.
